pkg/network/dns/resolveconf_test.go

```diff
@@ -18,6 +18,14 @@ var _ = Describe("Resolveconf", func() {
 			Expect(err).To(BeNil())
 		})
 
+		It("should allow parsing of IPv4 nameservers with leading zeros", func() {
+			ns1, ns2 := []uint8{8, 8, 8, 8}, []uint8{8, 8, 4, 4}
+			resolvConf := "nameserver 08.8.8.08\nnameserver 8.08.04.4\n"
+			nameservers, err := ParseNameservers(resolvConf)
+			Expect(nameservers).To(Equal([][]uint8{ns1, ns2}))
+			Expect(err).To(BeNil())
+		})
```

Comment:
I am aware that you have just copy/paste it from the previous test, but:
- Please separate between expectations and the input data.
- There is no need to check the error independently, this will also work:
`Expect(ParseNameservers(resolvConf)).To(Equal([][]uint8{ns1, ns2}))`

Reply:
This is not copy/paste, I've used an existing test and added leading zeros to the input fields.
I prefer to make this change in another PR.

Reply:
Done.